src/bp/core/misc/interfaces.ts

```diff
@@ -58,6 +58,7 @@ export interface AuthUser {
   last_logon?: Date
   created_on?: Date
   password_expired?: boolean
+  isSuperAdmin?: boolean
```

Comment:
If this field is to be temporary, either it should be added in its own typing (eg: requestauthuser) or it should be documented that it's temporary and can't be saved as a user property.